uhubctl.c

```diff
@@ -198,6 +199,7 @@ static int hub_phys_count = 0;
 /* default options */
 static char opt_vendor[16]   = "";
 static char opt_location[32] = "";     /* Hub location a-b.c.d */
+static int opt_level = -1;
```

Comment:
Perhaps use `0` for any level instead? Also add comment to explain what level means.